.bots/header-checker-lint.json

```diff
@@ -0,0 +1,5 @@
+{
+    "sourceFileExtensions": ["js", "py"],
+    "allowedLicenses": ["Apache-2.0"],
+    "ignoreFiles": ["**/node_modules/*"]
+}
```

Comment:
I think the bot only reads the configuration file from the master branch. Can you first add this file, then this PR?

Reply:
Also, they seems to prefer `.github/header-checker-lint.yml`.

Reply:
Done - PTAL?

https://github.com/GoogleCloudPlatform/repo-automation-playground/pull/36

Reply:
Can you remove this file?

Reply:
Done.